sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -1654,8 +1691,8 @@ object RapidsConf {
       // scalastyle:on line.size.limit
 
       println("\n## General Configuration\n")
-      println("Name | Description | Default Value")
-      println("-----|-------------|--------------")
+      println("Name | Description | Default Value | Applicable at")
```

Comment:
in the usage statement above:

All configs can be set on startup, but some configs, especially for shuffle, will not
        | work if they are set at runtime.
        
        Wonder if we add something about checking the table

Reply:
Added some text to explain the meaning of "Applicable at".

Reply:
@tgravescs Is it ok to resolve conversation?